include/ck_tile/ops/fmha/pipeline/block_fmha_pipeline_enum.hpp

```diff
@@ -10,7 +10,7 @@ enum class BlockFmhaPipelineEnum
 {
     QRKSVS = 0,
     QRKSVS_ASYNC,
-    QRKSVS_FP8,
+    // QRKSVS_FP8,
```

Comment:
you can just remove thos enumerator, since we already mark the pipeline `[[deprecated]]`. so there is no way to use it